tests/rip7560/validation_test.go

```diff
@@ -0,0 +1,83 @@
+package rip7560
+
+import (
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/tests"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+func TestValidation_OOG(t *testing.T) {
+	validatePhase(newTestContext(t), types.Rip7560AccountAbstractionTx{
+		ValidationGas: uint64(1000),
+		GasFeeCap:     big.NewInt(1000000000),
+	}, "out of gas")
+}
+
+func TestValidation_ok(t *testing.T) {
+	validatePhase(newTestContext(t), types.Rip7560AccountAbstractionTx{
+		ValidationGas: uint64(1000000000),
+		GasFeeCap:     big.NewInt(1000000000),
+	}, "")
```

Comment:
P.S. Checking "error is an empty string" is not pretty, should check there is no error IMO.